docs/how-to/resize-jenkins-storage.md

```diff
@@ -0,0 +1,62 @@
+# How to resize the jenkins-home storage volume
+The default size of the jenkins-home storage volume for a fresh installation is 1GB. While this works for most scenarios, operators might need to have more storage for installing plugins, storing artifacts, and runninng builds/checking out SCMs on the built-in node.
+
+A low disk-space on the built-in node will cause the node to go offline, blocking jenkins from running jobs.
+
+## Create a backup
```

Comment:
Perhaps this should be its own how to which can be referenced from here?

Reply:
And perhaps we could make this easier by adding an action for backup?

Reply:
Also we should also think whether we would also need a `restore` action as well.

Reply:
Let's do that in a separate PR and update the docs once we have that 🙂 It would be great if we add the s3 integration and manage it all on the user's behalf